mne/io/eeglab/eeglab.py

```diff
@@ -68,10 +70,10 @@ def _check_eeglab_fname(fname, dataname):
     return data_fname
 
 
-def _check_load_mat(fname, uint16_codec):
+def _check_load_mat(fname, uint16_codec, preload=False):
```

Comment:
Want to add these where we can going forward
```suggestion
def _check_load_mat(fname, uint16_codec, *, preload=False):
```

Reply:
done